llmfoundry/data/packing.py

```diff
@@ -358,6 +364,9 @@ def auto_packing_ratio(
     tokenizer: PreTrainedTokenizerBase,
     device_batch_size: int,
     num_packing_ratios: int = 20,
+    max_ratio: float = 0,
```

Comment:
We shouldn't use actual numbers like 0 as a placeholder for something else. 
```suggestion
    max_ratio: Optional[float] = None,
```